### PR TITLE
Add Gaussian, TriaxialGaussian, and AxisymmetricGaussian potentials

### PR DESCRIPTION
**Pull request by prashjet**
Adds three new potential classes:

- `GaussianPotential`: spherical Gaussian density profile, closed-form via erf.
- `TriaxialGaussianPotential`: Chandrasekhar ellipsoidal quadrature, mirroring the NFW/TriaxialNFW structure.
- `AxisymmetricGaussianPotential`: the q1=1 special case of `TriaxialGaussianPotential` as its own class (single shape parameter q2), for API clarity in the common oblate/prolate case.

All three are cross-checked against galpy in tests; `AxisymmetricGaussianPotential` is additionally cross-checked against `TriaxialGaussianPotential(q1=1)`. Profiling showed no measurable speedup for the axisymmetric class over the triaxial one with q1 fixed — it earns its keep on API ergonomics alone.

**Note:** this overlaps with #756, which adds a spherical `GaussianPotential`. This PR additionally adds triaxial and axisymmetric variants under a `gaussian/` package rather than a single module. Happy to rebase on #756 or coordinate however maintainers prefer.

### CODE REVIEW

File: src/galax/potential/_src/builtin/gaussian/axisymmetric.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+from xmmutablemap import ImmutableMap
+
+import galax._custom_types as gt
+from galax.potential._src.base import default_constants
+from galax.potential._src.base_single import AbstractSinglePotential
+from galax.potential._src.params.base import AbstractParameter
+from galax.potential._src.params.field import ParameterField
+
+
+class GaussLegendreIntegrator(eqx.Module):  # type: ignore[misc]
```
Comment (nstarman) [inline src/galax/potential/_src/builtin/gaussian/axisymmetric.py]: There's a GaussLegendreIntegrator in https://github.com/GalacticDynamics/galax/blob/main/src/galax/potential/_src/builtin/nfw/triaxial.py. 
Maybe that can be moved into a common location and used here as well.

File: src/galax/potential/_src/builtin/gaussian/axisymmetric.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+
+    Setting up the potential still requires a 1D quadrature: unlike the
+    spherical case (`~galax.potential.GaussianPotential`), a non-spherical
+    density does not obey Newton's shell theorem, so there is no simple
+    closed form here either -- only a handful of specific density profiles
+    (Miyamoto-Nagai, Kuzmin, Satoh, ...) were designed to admit one, and the
+    Gaussian is not among them.
+    """
+
+    m: AbstractParameter = ParameterField(  # type: ignore[assignment]
```
Comment (nstarman) [inline src/galax/potential/_src/builtin/gaussian/axisymmetric.py]: The Gaussian profile has a finite total mass, so this parameter should be `m_tot`.

File: src/galax/potential/_src/builtin/gaussian/base.py [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+
+    $$ \Phi(r) = -\frac{G m}{r} \mathrm{erf}\left(\frac{r}{\sqrt{2} r_s}\right)
+    $$
+
+    Which has the expected behavior of being finite at the center and going to
+    the Kepler potential :math:`-G m / r` at large radii.
+
+    """
+
+    m: AbstractParameter = ParameterField(  # type: ignore[assignment]
```
Comment (nstarman) [inline src/galax/potential/_src/builtin/gaussian/base.py]: The Gaussian profile has a finite total mass, so this parameter should be `m_tot`.

File: src/galax/potential/_src/builtin/gaussian/triaxial.py [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+from xmmutablemap import ImmutableMap
+
+import galax._custom_types as gt
+from galax.potential._src.base import default_constants
+from galax.potential._src.base_single import AbstractSinglePotential
+from galax.potential._src.params.base import AbstractParameter
+from galax.potential._src.params.field import ParameterField
+
+
+class GaussLegendreIntegrator(eqx.Module):  # type: ignore[misc]
```
Comment (nstarman) [inline src/galax/potential/_src/builtin/gaussian/triaxial.py]: This can be consolidated with the other GaussLegendreIntegrator

File: src/galax/potential/_src/builtin/gaussian/triaxial.py [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+        r_s^2}\right)
+
+    where
+
+    .. math::
+
+        \xi^2 = x^2 + \frac{y^2}{q_1^2} + \frac{z^2}{q_2^2}
+    """
+
+    m: AbstractParameter = ParameterField(  # type: ignore[assignment]
```
Comment (nstarman) [inline src/galax/potential/_src/builtin/gaussian/triaxial.py]: The Gaussian profile has a finite total mass, so this parameter should be `m_tot`.

File: src/galax/potential/_src/builtin/gaussian/axisymmetric.py [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+        ],
+        /,
+    ) -> Shaped[Array, "*batch"] | Shaped[u.Quantity["dimensionless"], "*batch"]:
+        y = f(self.x)
+        w = self.w.reshape(self.w.shape + (1,) * (y.ndim - 1))
+        return jnp.sum(y * w, axis=0)
+
+
+@final
+class AxisymmetricGaussianPotential(AbstractSinglePotential):
```
Comment (nstarman) [inline src/galax/potential/_src/builtin/gaussian/axisymmetric.py:29]: Maybe define an `AbstractGaussianPotential` ?

File: src/galax/potential/_src/builtin/gaussian/base.py [hunk 7]
```diff
@@ -0,0 +1,250 @@
...
+    """
+    x = r / p["r_s"]
+    m = p["m"]
+    erf_term = jsp.erf(x / jnp.sqrt(2.0))
+    exp_term = jnp.sqrt(2.0 / jnp.pi) * x * jnp.exp(-(x**2) / 2)
+    return m * (erf_term - exp_term)
+
+
+@ft.partial(jax.jit)
+def potential(p: gt.Params, r: gt.BBtSz0, /) -> gt.BtFloatSz0:
```
Comment (nstarman) [inline src/galax/potential/_src/builtin/gaussian/base.py:239]: This looks like it returns NaN at the origin.

File: src/galax/potential/_src/builtin/gaussian/base.py [hunk 8]
```diff
@@ -0,0 +1,250 @@
...
+    $$ \Phi(r) = -\frac{G m}{r} \mathrm{erf}\left(\frac{r}{\sqrt{2}
+    r_s}\right) $$
+
+    where $m$ is the total mass and $r_s$ is the scale radius.
+
+    """
+    r_s = p["r_s"]
+    x = r / r_s
+    phi0 = -p["G"] * p["m"] / r
+    return phi0 * jsp.erf(x / jnp.sqrt(2.0))
```
[the remark this answers is not in the document]
Comment (prashjet) [reply src/galax/potential/_src/builtin/gaussian/base.py]: will address in next commit

File: src/galax/potential/_src/builtin/gaussian/triaxial.py [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+
+
+@final
+class TriaxialGaussianPotential(AbstractSinglePotential):
+    r"""Triaxial (density) Gaussian Potential.
+
+    .. math::
+
+        \rho(q) = \frac{G M}{(2\pi)^{3/2} r_s^3} \exp\left(-\frac{\xi^2}{2
+        r_s^2}\right)
```
[the remark this answers is not in the document]
Comment (prashjet) [reply src/galax/potential/_src/builtin/gaussian/triaxial.py]: will address in next commit

File: src/galax/potential/_src/builtin/gaussian/axisymmetric.py [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+    r"""Axisymmetric (oblate/prolate) Gaussian Potential.
+
+    .. math::
+
+        \rho(R, z) = \frac{G M}{(2\pi)^{3/2} r_s^3} \exp\left(-\frac{\xi^2}{2
+        r_s^2}\right)
```
[the remark this answers is not in the document]
Comment (prashjet) [reply src/galax/potential/_src/builtin/gaussian/axisymmetric.py]: will address in next commit

File: src/galax/potential/_src/builtin/gaussian/axisymmetric.py (hunk 2 above)
Comment (prashjet) [reply src/galax/potential/_src/builtin/gaussian/axisymmetric.py]: will address in next commit

File: src/galax/potential/_src/builtin/gaussian/triaxial.py (hunk 4 above)
Comment (prashjet) [reply src/galax/potential/_src/builtin/gaussian/triaxial.py]: will do - will move this to src/galax/potential/_src/utils.py

File: src/galax/potential/_src/builtin/gaussian/axisymmetric.py (hunk 6 above)
Comment (prashjet) [reply src/galax/potential/_src/builtin/gaussian/axisymmetric.py:29]: Good suggestion, but I'll hold off for now: there's no `AbstractNFWPotential` either, despite that family having just as many variants (`NFWPotential`, `TriaxialNFWPotential`, `LeeSutoTriaxialNFWPotential`, `HardCutoffNFWPotential`, `Vogelsberger08TriaxialNFWPotential`, `gNFWPotential`). Introducing `AbstractGaussianPotential` here would make this family inconsistent with the existing precedent rather than more consistent with it. Happy to do a follow-up PR that introduces the abstract base for both families together, if that's the direction maintainers want.

Comment (nstarman) [reply src/galax/potential/_src/builtin/gaussian/axisymmetric.py:29]: A good question. I held off on the NFWs mostly because of the Zhao superset. Resolving the relationships between potentials is challenging to fit in an inheritance chain.

**Comment by prashjet** [comment]
Thanks for the thorough review! Pushed two follow-up commits addressing the feedback:

**Mass parameter (@nstarman, Copilot):**
- Renamed `m` → `m_tot` on all three classes.
- For `TriaxialGaussianPotential`/`AxisymmetricGaussianPotential`, fixed the density normalization to divide by `q1*q2` (resp. `q2`), so `m_tot` is the true total mass for any flattening — not just in the spherical case. Previously the parameter silently meant "mass scale," matching the (infinite-mass) NFW convention, which is misleading for a profile with finite mass.

**`r=0` NaN (Copilot):**
- Fixed the `0/0` in `GaussianPotential.potential()` by substituting the analytic limit `Φ(0) = -G m_tot √(2/π) / r_s` explicitly. Note: `gradient`/`hessian` at exactly `r=0` are still `NaN` — that's a separate, pre-existing limitation shared by every radial potential in the codebase (e.g. `HernquistPotential` has the same issue), coming from `vector_norm` not being differentiable at the origin. Out of scope here.

**Stray `G` in density docstrings (Copilot):**
- Fixed in `TriaxialGaussianPotential`/`AxisymmetricGaussianPotential`.

**Duplicated `GaussLegendreIntegrator` (@nstarman):**
- Consolidated into `potential/_src/utils.py` alongside the other cross-potential helpers, with a `GaussLegendreIntegrator.for_order()` factory so each class's `__post_init__`/`integration_order` boilerplate collapses to one line. Used by `TriaxialNFWPotential` too now.

**`AbstractGaussianPotential` (@nstarman):** replied inline — holding off since there's no `AbstractNFWPotential` either despite that family having more variants; happy to do a follow-up covering both families if that's the preferred direction.

All existing + updated tests pass (1026 passed locally).

**Comment by nstarman** [comment]
@prashjet if you rebase on main it should fix the Format CI job.

File: src/galax/potential/__init__.py [hunk 11]
```diff
@@ -27,6 +27,9 @@
     "MultipoleInnerPotential",
     "MultipoleOuterPotential",
     "MultipolePotential",
+    "AxisymmetricGaussianPotential",
+    "GaussianPotential",
+    "TriaxialGaussianPotential",
```
[the remark this answers is not in the document]
Comment (prashjet) [reply src/galax/potential/__init__.py:32]: resolved in 9d0006b

**Review by nstarman — APPROVED** [review]
Thanks! @prashjet. LMK what other features would be useful for your project.
